sdk/communication/Azure.Communication.Common/tests/CommunicationRecordedTestSanitizer.cs

```diff
@@ -12,6 +12,7 @@ public class CommunicationRecordedTestSanitizer : RecordedTestSanitizer
     {
         private static readonly Regex _azureResourceRegEx = new Regex(@"[^/]+?(?=(.communication.azure))", RegexOptions.Compiled);
         private static readonly Regex _identityInRouteRegEx = new Regex(@"(?<=identities/)([^/]+)", RegexOptions.Compiled);
+        private static readonly Regex _turnInRouteRegEx = new Regex(@"(?<=turn/)([^/]+)", RegexOptions.Compiled);
```

Comment:
With the [change from earlier today](https://github.com/Azure/azure-sdk-for-net/pull/18715), this should be move to the sanitizer from the identity package.

Reply:
Updated to follow the new sanitizer rules.